build/ghostunnel/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.14-alpine as build
+
+ARG VERSION=v1.5.2
+ARG TARGETPLATFORM
+ENV GO111MODULE=on \
+    CGO_ENABLED=0
+RUN apk add --no-cache \
+        git \
+        gcc \
+        musl-dev \
+        libtool \
+        make
+WORKDIR /go/src/github.com/square/ghostunnel
+RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
+    export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) && \
+    GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3); export GOARM=${GOARM:1} && \
+    git clone --depth 1 -b ${VERSION} https://github.com/square/ghostunnel.git . && \
+    make clean ghostunnel && \
+    cp ghostunnel /usr/bin/ghostunnel
```

Comment:
Nice work, we can probably remove `cp ghostunnel /usr/bin/ghostunnel` and then we can update our copy command `COPY --from=build /go/src/github.com/square/ghostunnel/ghostunnel /usr/bin/ghostunnel` what you think?

Reply:
Thanks for feedback!